zucchini-ui-frontend/src/testRuns/components/TestRunsTableContainer.js

```diff
@@ -5,14 +5,26 @@ import TestRunsTable from "./TestRunsTable";
 
 import { selectLatestTestRuns } from "../selectors";
 
+import * as utils from "../../utils/testRunUtils";
+
 const selectTestRuns = createSelector(
   state => selectLatestTestRuns(state),
   (state, ownProps) => ownProps.selectedType,
-  (testRuns, selectedType) => {
-    if (selectedType) {
-      return testRuns.filter(testRun => testRun.type === selectedType);
+  (state, ownProps) => ownProps.selectedNom,
+  (testRuns, selectedType, selectedNom) => {
+    var selectedTestRuns = testRuns;
+    if (selectedType && selectedNom) {
+      //On filtre une première fois selon le type
+      selectedTestRuns = selectedTestRuns.filter(testRun => utils.getType(testRun.type) === selectedType);
+      //On filtre une première fois selon le nom
+      //selectedTestRuns = selectedTestRuns.filter(testRun => utils.getNom(testRun.nom,testRun.type) === selectedNom);
+      selectedTestRuns = selectedTestRuns.filter(testRun => utils.getType(testRun.type) === selectedType);
```

Comment:
Il y a une erreur. On filtre deux fois de suite en fonction du type au lieu de filtrer en fonction du nom pour la 2nde fois. Pourquoi ne pas tout faire en un seul filtre ?

Reply:
là c'est une erreur et un détritus. Dans l'existant, il y'a un filtre selon le "type" qui n'est pas réellement le type mais une concaténation des champs type, nom, et environnement. Comme expliqué dans la JIRA, on filtre selon le vrai type. Puis on a voulu ajouter un second filtre selon le nom. Sauf qu'on s'est rendu compte que ça ne servirait pas à grand chose
```suggestion
      selectedTestRuns = selectedTestRuns.filter(testRun => utils.getType(testRun.type) === selectedType);
```